hexrd/ui/calibration/panel_buffer_dialog.py

```diff
@@ -160,3 +161,12 @@ def mode(self, v):
     def update_mode_tab(self):
         mode_tab = getattr(self.ui, self.mode + '_tab')
         self.ui.tab_widget.setCurrentWidget(mode_tab)
+
+    def clear_panel_buffer(self):
+        # Clear the config options on the internal config
+        detector_config = HexrdConfig().detector(self.detector)
+
+        buffer_default = {'status': 0}
+        buffer = detector_config.setdefault('buffer', buffer_default)
+
+        buffer['value'] = buffer['value'] = [0., 0.]
```

Comment:
Uh oh, looks like `buffer['value'] =` is there twice?

Reply:
Fixed!